### DISCUSSION TITLE
Export File On Field Grid in Different Coordinate System

### DISCUSSION THREAD

**Discussion by RyanAshbaugh** [ansys/pyaedt#3982] [Q&A] [answered]
Basically what the title says, I want to use the `export_field_file_on_grid()` function to export a field from a coordinate system other than Global. From what I can tell this functionality is not supported. It would make things much easier as it also does not seem like the functionality transforming coordinates from 1 coordinate system to another is currently implemented.

**Comment by Samuelopez-ansys** [comment]
Hi @RyanAshbaugh ,

You are right, it has not been implemented yet in PyAEDT, please submit an issue, and the community will implement it. In the meantime, you can use the AEDT API as a workaround, because you can use PyAEDT commands and AEDT API commands in the same code.

```
hfss = pyaedt.Hfss()
oModule = hfss.odesign.GetModule("FieldsReporter")
oModule.EnterQty("E")

oModule.ExportOnGrid("C:\\test.fld", ["1mm", "1mm", "1mm"], ["2mm", "2mm", "2mm"], ["1mm", "1mm", "1mm"], "Setup1 : Field_Plot", 
	[
		"Freq:="		, "500MHz",
		"Phase:="		, "0deg"
	], 
	[
		"NAME:ExportOption",
		"IncludePtInOutput:="	, True,
		"RefCSName:="		, "RelativeCS1",
		"PtInSI:="		, True,
		"FieldInRefCS:="	, False
	], "Cartesian", ["0mm", "0mm", "0mm"], False)
```

**Reply by Samuelopez-ansys** [reply, marked as answer]
This method was implemented:

export_field_file_on_grid
